frontend/logout.ts

```diff
@@ -8,7 +8,12 @@ window.addEventListener('DOMContentLoaded', () => {
     }
 
     console.log("Logging out all chatrix sessions");
-    logoutAndDeleteData().catch(error => console.log(error));
+    logoutAndDeleteData().catch(error => console.log(error)).finally(() => {
+        // Logout has been done and data has been deleted.
+        // We can now expire the logout cookie.
+        const now = (new Date).toUTCString();
+        document.cookie = `${logoutCookieName}=false; expires=${now};path=/;`;
+    });
```

Comment:
```suggestion
    logoutAndDeleteData().then(() => {
        // Logout has been done and data has been deleted.
        // We can now expire the logout cookie.
        const now = (new Date).toUTCString();
        document.cookie = `${logoutCookieName}=false; expires=${now};path=/;`;
    }).catch(error => console.log(error));
```

Reply:
Better to not attempt to expire cookie when promise has been rejected, so it can be re-tried?

Reply:
I considered both cases, and in the end went with only trying it once. This was to prevent that we would perpetually be attempting logout. However, given that the cookie expires in 10 minutos, we would only be attempting for that duration. So I agree, it's best to not delete the cookie when an error occurs 👍